manifests/validator.pp

```diff
@@ -0,0 +1,25 @@
+# == Class: grafana::validator
+#
+# Includes `` resource
+#
+# === Parameters
+# [*grafana_url*]
```

Comment:
please convert this syntax to puppet-strings style

Reply:
Done